.github/workflows/uploaddockerimage.yml

```diff
@@ -0,0 +1,49 @@
+name: uploaddockerimage
+
+on:
+  workflow_call:
+    inputs:
+      platform:
+        required: true
+        type: string
+      labels:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+      registry:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+    secrets:
+      docker_repo_username:
+        required: true
+      docker_repo_password:
+        required: true
+
+jobs:
+  upload_job:
+    name: Upload docker image to quay
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_QUAY: ${{ inputs.registry }}/${{ inputs.repository }}
+    steps:
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ inputs.registry }}
+          username: ${{ secrets.docker_repo_username }}
+          password: ${{ secrets.docker_repo_password }}
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: operator-${{ inputs.platform }}
+          path: /tmp
+      - name: Upload image to Quay
+        run: |
+            docker load --input /tmp/operator-${{ inputs.platform }}.tar
+            docker tag operator-${{ inputs.platform }}:${{ inputs.version }} ${IMAGE_QUAY}:${{ inputs.version }}-${{ inputs.platform }}
```

Comment:
You could build the strings in the `env` section of this step, like `IMAGE_QUAY` a few steps before.
Would make it more explicit.